.goreleaser.yaml

```diff
@@ -0,0 +1,43 @@
+before:
+  hooks:
+    - go mod download
+
+builds:
+- id: "apko-build"
+  binary: apko
+  main: ./cmd/apko/main.go
+  env:
+  - CGO_ENABLED=0
+  goos:
+    # apko requires alpine, so only build linux binaries.
+    - linux
+  goarch:
+    - 386
+    - amd64
+    - arm64
+  hooks:
+    post:
+      - sh -c "COSIGN_EXPERIMENTAL=true cosign sign-blob --output-certificate dist/apko_{{ .Version }}_{{ .Os }}_{{ .Arch }}.crt --output-signature dist/apko_{{ .Version }}_{{ .Os }}_{{ .Arch }}.sig {{ .Path }}"
```

Comment:
we can use the sign step for that, if you don't mind I can open a follow up to update